pkg/loop/server.go

```diff
@@ -68,18 +72,41 @@ func (s *Server) start() error {
 		return fmt.Errorf("error getting environment configuration: %w", err)
 	}
 
-	if err := SetupTracing(TracingConfig{
+	tracingConfig := TracingConfig{
 		Enabled:         envCfg.TracingEnabled,
 		CollectorTarget: envCfg.TracingCollectorTarget,
 		SamplingRatio:   envCfg.TracingSamplingRatio,
 		TLSCertPath:     envCfg.TracingTLSCertPath,
 		NodeAttributes:  envCfg.TracingAttributes,
 		OnDialError:     func(err error) { s.Logger.Errorw("Failed to dial", "err", err) },
-	}); err != nil {
+	}
+
```

Comment:
I need to add back the old path here, for cases where Beholder is not configured.